RELEASE/scripts/autoscend/auto_quest_level_13.ash

```diff
@@ -1,5 +1,214 @@
 script "auto_quest_level_13.ash"
 
+boolean needStarKey()
+{
+	if(contains_text(get_property("nsTowerDoorKeysUsed"),"star key"))
+	{
+		return false;
+	}
+	if(item_amount($item[Richard\'s Star Key]) > 0)
+	{
+		return false;
+	}
+	return true;
+}
+
+boolean needDigitalKey()
+{
+	if(contains_text(get_property("nsTowerDoorKeysUsed"),"digital key"))
+	{
+		return false;
+	}
+	if(item_amount($item[Digital Key]) > 0)
+	{
+		return false;
+	}
+	if(whitePixelCount() >= 30)
+	{
+		return false;
+	}
+
+	return true;
+}
+
+int whitePixelCount()
+{
+	int count = item_amount($item[White Pixel]);
+
+	int extra = min(item_amount($item[Red Pixel]), item_amount($item[Blue Pixel]));
+	extra = min(extra, item_amount($item[Green Pixel]));
+	return count + extra;
```

Comment:
This is fine but a quicker way to do this would be `return item_amount($item[White Pixel]) + creatable_amount($item[White Pixel]);`

Reply:
change made.

Reply:
Also tested via test script and can confirm it works correctly